plugins/network-elements/nuage-vsp/src/com/cloud/network/guru/NuageVspGuestNetworkGuru.java

```diff
@@ -114,7 +115,9 @@ public Network design(NetworkOffering offering, DeploymentPlan plan, Network use
         PhysicalNetworkVO physnet = _physicalNetworkDao.findById(plan.getPhysicalNetworkId());
         DataCenter dc = _dcDao.findById(plan.getDataCenterId());
         if (!canHandle(offering, dc.getNetworkType(), physnet)) {
-            s_logger.debug("Refusing to design this network");
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("Refusing to design network using network offering " +  offering.getId() + (physnet != null ? " on physical network " + physnet.getId() : ""));
```

Comment:
Not really, all plugins are logging this as <code>DEBUG</code>. 
The code that's calling this method is as following
```java
            for (final NetworkGuru guru : networkGurus) {
                final Network network = guru.design(offering, plan, predefined, owner);
                if (network == null) {
                    continue;
                }
                ...
            }
```
It's looping over all gurus, we don't want our guru to log it as <code>WARN</code> every time a network is created and we don't support it.